site/content/docs/getting-started/03-containers/index.md

```diff
@@ -0,0 +1,241 @@
+            right: 10.,
+            top: 10.,
+            bottom: 10.,
+        },
+        // Use the default value for the rest of the layout options
+        ..Default::default()
+    });
+
+    widget! {
+        // Our content box
+        (content_box [ // <---- Note the square bracket here used to specify children of
+                       //       the content box
+
+            // Our text widget is now _inside_ of the content box
+            (text_box: {text_box_props})
+        ])
+    }
+}
+```
+
+{% end %}
+
+Now if we run our app we should have a small margin between the text and the sides of the window.
+
+But what if we want to add another item to our box, like maybe an image? Let's see what happens if we place an image in the content box with our text:
+
+{% rustdoc_test() %}
+
+```rust
+# use raui::prelude::*;
+pub fn app(_ctx: WidgetContext) -> WidgetNode {
+    // Create our text box properties
+    // ...
+#    let text_box_props = Props::new(TextBoxProps {
+#        text: "Hello World!".into(),
+#        color: Color {
+#            r: 0.0,
+#            g: 0.0,
+#            b: 0.0,
+#            a: 1.0,
+#        },
+#        font: TextBoxFont {
+#            // We specify the path to our font
+#            name: "resources/verdana.ttf".to_owned(),
+#            size: 60.0,
+#        },
+#        // Use the defaults for the rest of the text box settings
+#        ..Default::default()
+#    })
+#    .with(ContentBoxItemLayout {
+#        // Specify a margin of 10 on every side
+#        margin: Rect {
+#            left: 10.,
+#            right: 10.,
+#            top: 10.,
+#            bottom: 10.,
+#        },
+#        // Use the default value for the rest of the layout options
+#        ..Default::default()
+#    });
+
+    // Create the props for our image
+    let image_box_props = Props::new(ImageBoxProps {
+        // The material defines what image or color to use for the box
+        material: ImageBoxMaterial::Image(ImageBoxImage {
+            // The path to our image
+            id: "resources/cats.jpg".to_owned(),
+            ..Default::default()
+        }),
+        // Have the image fill it's container
+        width: ImageBoxSizeValue::Fill,
+        height: ImageBoxSizeValue::Fill,
+        ..Default::default()
+    });
+
+    widget! {
+        (content_box [
+            // Add our image box above our text box, that will make sure that the
+            // image appears _under_ our text box when the UI is rendered
+            (image_box: {image_box_props})
+            (text_box: {text_box_props})
+        ])
+    }
+}
+```
+
+{% end %}
+
+![text and image](text-and-image.png)
+
+> **Note:** You can download the demo cat image [here](https://github.com/PsichiX/raui/raw/master/site/rust/guide_03/resources/cats.jpg).
+
+Notice that the content box didn't make sure the was any "room" for the text or the image to sit side-by-side with each-other, it just stacked them right on top of each-other. Also
```

Comment:
typo i believe:
Notice that the content box didn't make sure **_there_** was any "room" for the text